### PR TITLE
formatted docstring of sympy.geometry.curve according to documentation.

### PR DESCRIPTION
**Pull request by viratkohli1969**
#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/geometry/curve.py [hunk 1]
```diff
@@ -298,14 +304,34 @@ def plot_interval(self, parameter='t'):
...
+
+        limits : Returns limits of the parameter interval
+
         """
         t = _symbol(parameter, self.parameter, real=True)
         return [t] + list(self.limits[1:])
 
     def rotate(self, angle=0, pt=None):
-        """Rotate ``angle`` radians counterclockwise about Point ``pt``.
+        """This function is used to rotate a curve along given point.
```
Comment (oscarbenjamin) [inline sympy/geometry/curve.py:317]: This explanation does not seem equivalent to the previous one

**Comment by oscarbenjamin** [comment]
I left one comment but otherwise this looks good to me. I don't know this part of the codebase so well though...

**Comment by friyaz** [comment]
@mohitshah3111999 I think we should just use 'NO ENTRY` for the release notes as the changes are related to documentation and are not major. See [Writing release notes](https://github.com/sympy/sympy/wiki/Writing-Release-Notes).

**Comment by mohitshah3111999** [comment]
> 
> 
> @mohitshah3111999 I think we should just use `NO ENTRY` for the release notes as the changes are related to documentation and are not major. See [Writing release notes](https://github.com/sympy/sympy/wiki/Writing-Release-Notes).

Yes you are right.
Actually I added this note because when I added my last pull request, At that time it has no issue. So I thought to add release note.

**Pull request closed without merging**